changelog/_unreleased/2021-09-04-use-question-helper-for-passwords-in-commands.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Use question helper for passwords in commands
+author: mynameisbogdan
+author_email: mynameisbogdan@protonmail.com
```

Comment:
Oh and it would be nice if you could add 'issue: NEXT-17106``  after the author line